src/wizard.ts

```diff
@@ -113,6 +118,12 @@ export class WizardBase {
         this.output.appendLine(text);
     }
 
+    protected onRunError(step: WizardStep, stepIndex: number, error: Error) {
+        if (!(error instanceof UserCancelledError)) {
+            vscode.window.showErrorMessage(error.message);
```

Comment:
Sometimes the message can be very long (for example, the Kudu server can return an entire HTML page of 4xx or 5xx error). Not sure in this case the showErrorMessage will be the best experience.

Reply:
What do you suggest?

Reply:
How about send this to the output channel instead? At least the full message can be seen. HTML should be readable to developers. Not saying this is a perfect solution, I'll create an issue with Kudu to ask for better server side error messages, for example, in JSON instead of HTML.

Reply:
As discussed, this would be during the wizard UI interaction, thus the warning bar seemed more appropriate here.